### DISCUSSION TITLE
Post instalation efi

### DISCUSSION THREAD

**Discussion by mikeyr0cks** [stevedat/HP-Elitebook-830-G6-Hackintosh#16] [Q&A]
Hi Steve! Can you please give me the post instalation EFI? thank you so much for your work!

**Comment by stevedat** [comment]
> Hi Steve! Can you please give me the post instalation EFI? thank you so much for your work!

Give me your email, I will share it to you.

**Comment by stevedat** [comment]
I am out of hackintosh, but remaining SSD with the hackintosh if you like it.

**Comment by mikeyr0cks** [comment]
HI! Thanks a lot!! mihaita162002@gmail.com

**Comment by mikeyr0cks** [comment]
Also, I'm having some weird issues. When I press the restart button, the os freezes and needs to be forcefully closed. That also happens when I use the system after some time or if I leave it unused for 5-10 minutes. It is very weird.

**Comment by mikeyr0cks** [comment]
Hi again Steve! Can you please send me the post install EFI? Im having trouble with the the hackintosh, it keeps freezing or it freezes when i hit shut down or reboot! Thanks!
